### ISSUE TITLE
Diff not showing show-skipped info when using compare-to

### ISSUE THREAD

**Issue by RaZloR** [infracost/infracost#2233]
Hello Team,

I don't know if i'm doing something wrong here...

Im using multi folders with a config file, comparing a feature branch with the main branch and posting a comment in a DevOps PR.
The output shows, as expected, the cost difference inside the feature itself (columns new and diff if its new module and new, previous and diff if modifying a module) and the cost of all modules (projects).
The report seems OK, and although both baseline and target branches breakdowns have the show-skipped info, the diff does not show the show-skipped data.

Is this expected or am i doing something wrong here?

Ex: infracost diff --path "VM-feature.json" --compare-to "VM-baseline.json" --show-skipped

Azure DevOps
yaml with infracost CLI tasks
Config-file with multi folders

Thank you for your help!

**Comment by alikhajeh1** [comment]
Thanks @RaZloR! I've confirmed that's a bug and added reproduction steps below. 

```
$ git clone https://github.com/infracost/example-terraform

$ cd example-terraform

$ infracost breakdown --path . --format json --out-file infracost-baseline.json

$ vi main.tf and add this resource:

resource "google_apigee_instance" "apigee_instance" {
  name     = "tf-test"
  location = "us-central1"
  org_id   = "123"
}

$ infracost breakdown --path . --format json --out-file infracost.json

$ cat infracost.json | jq '.summary.unsupportedResourceCounts' 
# NOTE that the unsupported resource is shown in the file.

$ infracost diff --path infracost.json --compare-to infracost-baseline.json --show-skipped 
# NOTE the skipped resource is not shown.
#      the skipped resource should be shown, similar to what `infracost output --path infracost.json --show-skipped` does

```

**Comment by RaZloR** [comment]
Thank you @alikhajeh1 for your fast reply!
Looking forward for a fix. 

Thank you

**Comment by tim775** [comment]
@RaZloR Thanks for the report, the fix will go out in the 0.10.17 release.

**Comment by hugorut** [comment]
Hello @RaZloR the fix for this issue has been released as part of the [v0.10.17](https://github.com/infracost/infracost/releases/tag/v0.10.17) infracost CLI. Thanks for bringing this to our attention.
